### PR TITLE
Docker integration

### PR DESCRIPTION
**Pull request by dolgopolovwork**
Chain-adapter Docker integration.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Please run `kscript --package grpc_healthcheck.kts` before building docker image
+COPY grpc_healthcheck /opt/chain-adapter
+
+## Wait for script (see https://github.com/ufoscout/docker-compose-wait/)
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.5.0/wait /opt/chain-adapter/wait
+RUN chmod +x /opt/chain-adapter/wait
+
+COPY entrypoint.sh /opt/chain-adapter/
+RUN chmod +x /opt/chain-adapter/entrypoint.sh
+ENTRYPOINT /opt/chain-adapter/wait && /opt/chain-adapter/entrypoint.sh
```
Comment (Alexey-N-Chernyshov) [inline Dockerfile]: exec form is preferred.

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+STATE=1
+
+while [ $STATE -ne 0 ]; do
+    ./grpc_healthcheck ${IROHA_HOST} ${IROHA_PORT}
+    STATE=$?
+    echo "Waiting for iroha on ${IROHA_HOST}:${IROHA_PORT}"
+    sleep 0.1
+done
+
+java -jar /opt/chain-adapter/chain-adapter.jar
```
Comment (Alexey-N-Chernyshov) [inline entrypoint.sh]: exec java -jar /opt/chain-adapter/chain-adapter.jar
to replace shell process

File: entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if  [ -z "${IROHA_HOST}" ] || [ -z "${IROHA_PORT}" ]; then
+    echo "Please set IROHA_HOST and IROHA_PORT variables"
+    exit
+fi
+
+STATE=1
+
+while [ $STATE -ne 0 ]; do
```
Comment (Mingela) [inline entrypoint.sh]: you do it by 'docker-compose-wait' no need to do it again manually

Comment (dolgopolovwork) [reply entrypoint.sh]: This code is simply copied from D3 repo. I don't even know what the script does.

**Pull request merged**